books/RayTracingInOneWeekend.html

```diff
@@ -524,66 +525,66 @@
 Adding a Sphere
 ====================================================================================================
 
-<div class='together'>
 Let’s add a single object to our ray tracer. People often use spheres in ray tracers because
 calculating whether a ray hits a sphere is pretty straightforward.
 
 
 Ray-Sphere Intersection
 ------------------------
+<div class='together'>
 Recall that the equation for a sphere centered at the origin of radius $R$ is $x^2 + y^2 + z^2 =
 R^2$. Put another way, if a given point $(x,y,z)$ is on the sphere, then $x^2 + y^2 + z^2 = R^2$. If
 the given point $(x,y,z)$ is _inside_ the sphere, then $x^2 + y^2 + z^2 < R^2$, and if a given point
 $(x,y,z)$ is _outside_ the sphere, then $x^2 + y^2 + z^2 > R^2$.
 
-It gets uglier if the sphere center is at $(\mathbf{c}_x, \mathbf{c}_y, \mathbf{c}_z)$:
+It gets uglier if the sphere center is at $(\mathbf{C}_x, \mathbf{C}_y, \mathbf{C}_z)$:
```

Comment:
I don't know if we discussed this, but I think components of a vector should be stylized as scalars, i.e.
`$(c_x, c_y, c_z)$`

Not sure if they should be upper case or lower case: `$C_x%` or `$c_x$`

Reply:
Agreed — this is a mistake. Will fix.

Reply:
I've kept the scalar components with the same case as their source. Vector components are lowercase, point components are uppercase.